doc/HOWTO_RELEASE.rst.txt

```diff
@@ -303,14 +319,41 @@ Identify the commit hash of the release, e.g. 1b2e1d63ff.
 ::
     git co 1b2e1d63ff # gives warning about detached head
 
-Now, set ``release=True`` in setup.py, then
+First, change/check the following variables in ``pavement.py`` depending on the
+release version::
 
-::
+    RELEASE_NOTES = 'doc/release/1.7.0-notes.rst'
+    LOG_START = 'v1.6.0'
+    LOG_END = 'maintenance/1.7.x'
+
+Do any other changes. When you are ready to release, do the following
+changes::
 
-    git commit -m "REL: Release." setup.py
+    diff --git a/setup.py b/setup.py
+    index b1f53e3..8b36dbe 100755
+    --- a/setup.py
+    +++ b/setup.py
+    @@ -57,7 +57,7 @@ PLATFORMS           = ["Windows", "Linux", "Solaris", "Mac OS-
+     MAJOR               = 1
+     MINOR               = 7
+     MICRO               = 0
+    -ISRELEASED          = False
+    +ISRELEASED          = True
+     VERSION             = '%d.%d.%drc1' % (MAJOR, MINOR, MICRO)
+
+     # Return the git revision as a string
+
+And make sure the ``VERSION`` variable is set properly.
+
+Now you can make the release commit::
+
+    git commit -s -m "REL: Release." setup.py
     git tag <version>
     git push origin <version>
```

Comment:
Should also be clear what origin refers to here. For me it is my numpy fork on github.